backend/ibutsu_server/controllers/admin/project_controller.py

```diff
@@ -87,10 +88,11 @@ def admin_get_project_list(
     if group_id:
         query = query.filter(Project.group_id == group_id)
 
-    offset = (page * page_size) - page_size
-    offset = 0 if offset < 0 else offset
+    offset = get_offset(page, page_size)
     total_items = query.count()
     total_pages = (total_items // page_size) + (1 if total_items % page_size > 0 else 0)
+    if offset > 9223372036854775807:  # max value of bigint
```

Comment:
Is there a constant somewhere with this value?

Reply:
oh yeah, good point! I found it in `sys.maxsize`